src/main.ts

```diff
@@ -15,7 +15,7 @@ async function bootstrap() {
   const adapterHost = app.get(HttpAdapterHost);
   app.useGlobalFilters(new GlobalExceptionFilter(adapterHost));
 
-  const port = process.env.PORT || 3333;
+  const port = process.env.PORT || 3334;
```

Comment:
Don't modify this, I don't think is part of the task

Reply:
That's because of my Local Env. Throw me an error if I use the 3333 port

Reply:
If the probel is local, so you should not update anything in the server